docs/geti_tune/entities.md

```diff
@@ -0,0 +1,45 @@
+# Geti Tune entities
+
+![Full ER diagram with all entities](media/er/full.svg)
+
+In Geti Tune, a _project_ (`Project`) represents the machine learning problem that the user wants to solve,
+and at the same time serves as a container for all the assets related to it (datasets, models, ...)
+
+The definition of the problem consists of a _task_ and a set of _labels_ (`Label`).
+The `LabelSchema` entity encapsulates the information about the task type, the labels and their relationships.
+An example of a task is "object detection" with labels "car", "person" and "bicycle".
+
+Each project has an `InferencePipeline`, that is a sequence of processing steps that acquires a stream of
+data from a _source_ (`Source`), runs inference using a _model_ (`ModelRevision`) like YOLO, processes the results 
+and eventually sends them to a _sink_ (`Sink`). Examples of sources are video files or IP cameras, while examples of
+sinks are folders on the local filesystem or messaging frameworks like MQTT or ROS2.
+After inference, the frames and their predictions are analyzed to identify potential anomalies or out-of-distribution
+samples (_model monitoring_), and these may be collected based on one or more policies (`DataCollectionPolicy`).
+
+The collected frames are stored in a _dataset_ (`Dataset`), These frames are saved in a dataset, so that the user can
+examine them and eventually use them, for example, to make the model more robust via fine-tuning.
+It is also possible for the user to manually upload images to a dataset, and annotate them using the built-in
+annotation tool. A dataset is therefore a collection of image-like items (`DatasetItem`), each of which may have
+some _annotation_ data (`Annotation`) that is either a _user annotation_ or a _model prediction_.
+
+In addition to annotations, which are used when training and evaluating models, it is also possible to assign
+custom _tags_ to each media. A tag is a simple label that has no effect on model training or evaluation, but can be
+used to organize and filter the dataset items. Filtered data, based on tags and/or other properties (e.g. timestamp),
+can be used to create a _view_ in the dataset (`DatasetView`). A dataset view basically represents a subset of the
```

Comment:
Can you elaborate on what the intention of a `DatasetView` is? Is this similar to the `POST dataset:query` endpoint we have at Geti?

Reply:
This PR is still a draft, I'm rewriting the datasets document.

In short, the dataset view is a custom subset of the dataset. The user can create views with any items he wants, and he can use views as input for other workflows (evaluation, export, ...). Views can be created through a filtering query, but it is also possible to assign/remove items by id.